{"frames":[{"func":"write.csv(df4b, file = \"df4b.csv\")","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"eval.parent(Call)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"eval(expr, p)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"eval(expr, p)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"utils::write.table(df4b, file = \"df4b.csv\", col.names = NA, sep = \",\", \n    dec = \".\", qmethod = \"double\")","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"is.data.frame(x)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0}],"message":"Error en eval(expr, p): objeto 'df4b' no encontrado\n"}